kafka-client/src/main/java/dev/responsive/kafka/internal/stores/ResponsiveStoreRegistry.java

```diff
@@ -68,6 +68,12 @@ public synchronized List<ResponsiveStoreRegistration> getRegisteredStoresForChan
           "there should always be a store for the thread (%s) if there are stores registered "
               + "for this topic partition (%s)", threadId, topicPartition));
     }
+    if (storesForThread.size() > 1) {
+      LOGGER.warn("found more than 1 registration: {}",
```

Comment:
probably already in each registration but let's just add `found more than 1 registration for {threadId}: {}"`